src/site/content/en/blog/2022-state-of-css/index.md

```diff
@@ -0,0 +1,2008 @@
+---
+layout: post
+title: State of CSS 2022
+subhead: Web styling features of today and tomorrow, as seen at Google IO 2022, plus some extras.
+authors:
+  - adamargyle
+description: Web styling features of today and tomorrow, as seen at Google IO 2022, plus some extras.
+date: 2022-05-11
+thumbnail: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/iOfuipzyniP6aec4cOss.png
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/uT454OnumDCJbV0PWmvu.png
+alt: Colorful lettering using a Colr Font saying State of CSS 2022
+tags:
+  - blog
+  - css
+  - html
+---
+
+The year 2022 is set to be one of CSS's greatest years, in both features and
+cooperative browser feature releases, with a collaborative goal to implement 14
+features!
+
+## Overview
+
+This post is the article form of the talk given at Google IO 2022. It's not
```

Comment:
will link to the video (or embed it) once the link is ready. shouldnt be more than a couple days.

i should probably add the link the day of publish tho, so it's not leaked.

Reply:
@argyleink based on the email that I believe you were copied into, it looks like the resources will be live before the videos. So I intend to make this one, along with my article live early on the 11th. When the videos launch on the 12th I'll come back and make a PR to add them.

I think we could embed them?